assignments/objects.js

```diff
@@ -17,29 +17,76 @@ const example = {
   "gender": "F"
 }
 
-// Write your intern objects here:
 
 
+// Write your intern objects here:
+const companyList = [
+  {
+    "id": 1,
+    "name": "Mitzi",
+    "email": "mmelloy0@psu.edu",
+    "gender": "F"
+
+  },
+  KennanObj,
```

Comment:
JS will throw that KennanObj is undefined. I would not rely on hoisting so much in JS as it is mainly functions that get  hoisted and not variables. Quick fix is moving the KennanObj that was defined at the lower level before the companyList which will fix the issue.